Dockerfile

```diff
@@ -45,6 +45,7 @@ COPY --from=compile /dogstatsd_plugin.so .
 
 ENV PYTHONUNBUFFERED=1 \
     PORT=8888 \
+    KINTO_INI=/etc/kinto.ini \
```

Comment:
Any reason we don't want to set this to `config/example.ini`? Then we don't need the `mv` command below.